coordinator/src/db/user.rs

```diff
@@ -151,27 +151,14 @@ pub fn login_user(
 ) -> Result<()> {
     tracing::debug!(%trader_id, token, "Updating token for client.");
     let last_login = OffsetDateTime::now_utc();
-    let affected_rows = diesel::insert_into(users::table)
-        .values(NewUser {
-            id: None,
-            pubkey: trader_id.to_string(),
-            contact: "".to_owned(),
-            nickname: None,
-            timestamp: OffsetDateTime::now_utc(),
-            fcm_token: token.clone(),
-            version: version.clone(),
-            last_login,
-            // TODO: this breaks the used referral code
-            used_referral_code: None,
-        })
-        .on_conflict(schema::users::pubkey)
-        .do_update()
-        .set((
-            users::fcm_token.eq(&token),
-            users::last_login.eq(last_login),
-            users::version.eq(version),
-        ))
-        .execute(conn)?;
+    let affected_rows =
```

Comment:
The logic above would have done an update on conflict. Isn't it safer the way it was before?